src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -493,11 +545,14 @@ export const GenericTable = ({
         agTheme={context.isDarkTheme ? gridDark.agTheme : gridLight.agTheme}
         onImport={importFile}
         onExport={openExportDialog}
+        onAddRow={onAddRow}
+        onDeleteRow={onDeleteRow}
         onCellChange={onCellChange}
         onClearErrors={onClearErrors}
         buildErrorsPanelTitle={buildErrorsPanelTitle}
         maxErrorsCount={MAX_ERRORS_COUNT}
         isDirty={isDirty}
+        enableAddRow={enableAddRow}
```

Comment:
maybe, consider slightly renaming the option because `enableAddRow` suggests a function but it's a boolean (something like `isAddRowEnabled`, `addRowEnabled` or `isAddRowEnabled`)

Reply:
DONE